package.json

```diff
@@ -42,6 +42,9 @@
     "ts-loader": "^9.5.1",
     "typescript": "^5.3.3"
   },
+  "resolutions": {
+    "vscode-jsonrpc": "8.2.0"
```

Comment:
If you remove it and run a `yarn why vscode-jsonrpc` you'll see in the output that some other packages such as vscode-ws-jsonrpc or vscode-langaugeserver-protocol depend on it:

```
=> Found "vscode-jsonrpc@8.2.0"
info Has been hoisted to "vscode-jsonrpc"
info Reasons this module exists
   - "workspace-aggregator-46172982-3d9a-40ed-a50d-2815dae112fc" depends on it
   - Hoisted from "_project_#vscode-process-editor-webview#vscode-jsonrpc"
   - Hoisted from "_project_#designer-11#@eclipse-glsp#vscode-integration#vscode-jsonrpc"
   - Hoisted from "_project_#vscode-process-editor-webview#@axonivy#process-editor-protocol#@eclipse-glsp#protocol#vscode-jsonrpc"
   - Hoisted from "_project_#vscode-process-editor-webview#@axonivy#process-editor-inscription#@axonivy#inscription-core#vscode-jsonrpc"
   - Hoisted from "_project_#vscode-process-editor-webview#@axonivy#process-editor-inscription#@axonivy#inscription-core#vscode-languageserver-protocol#vscode-jsonrpc"
info Disk size without dependencies: "348KB"
info Disk size with unique dependencies: "348KB"
info Disk size with transitive dependencies: "348KB"
info Number of shared dependencies: 0

=> Found "vscode-ws-jsonrpc#vscode-jsonrpc@8.1.0"
info This module exists because "_project_#vscode-process-editor-webview#@axonivy#process-editor-inscription#@axonivy#inscription-core#vscode-ws-jsonrpc" depends on it.
info Disk size without dependencies: "348KB"
info Disk size with unique dependencies: "348KB"
info Disk size with transitive dependencies: "348KB"
info Number of shared dependencies: 0

=> Found "vscode-languageclient#vscode-jsonrpc@8.1.0"
info Reasons this module exists
   - "_project_#vscode-process-editor-webview#@axonivy#process-editor-inscription#@axonivy#inscription-core#monaco-languageclient#vscode-languageclient#vscode-languageserver-protocol" depends on it
   - Hoisted from "_project_#vscode-process-editor-webview#@axonivy#process-editor-inscription#@axonivy#inscription-core#monaco-languageclient#vscode-languageclient#vscode-languageserver-protocol#vscode-jsonrpc"
info Disk size without dependencies: "348KB"
info Disk size with unique dependencies: "348KB"
info Disk size with transitive dependencies: "348KB"
info Number of shared dependencies: 0
```